src/integration-test/java/developerTC/AgentService.java

```diff
@@ -30,17 +27,19 @@
 
     private static String invokeUrl;
 
-    private static String socketTimeout;
     private static String username;
     private static String password;
 
 
+
+    private static RemoteAccount remoteAccount;
+
+
     static {
         Properties properties = getProperties();
```

Comment:
Why are we doing all of this? Can we not use the Status API to connect via URL rather than manually constructing a DDO?

Reply:
Reverted the changes, will raise PR separately for a separate issue.